addon/components/o-s-s/power-select.ts

```diff
@@ -41,29 +41,32 @@ export default class OSSPowerSelect extends Component<OSSPowerSelectArgs> {
   }
 
   @action
-  observeResize(element: Element) {
+  observeResize(element: Element): void {
     this.resizeObserver.observe(element);
   }
 
   @action
-  unobserveResize(element: Element) {
+  unobserveResize(element: Element): void {
     this.resizeObserver.unobserve(element);
   }
 
   @action
-  ensureBlockPresence(name: string, hasNamedBock: boolean) {
-    if (!hasNamedBock) {
-      throw new Error(`[component][OSS::PowerSelect] You must pass ${name} named block`);
+  ensureBlockPresence(hasSelectedItem: boolean, hasOptionItem: boolean): void | never {
```

Comment:
I hadn't raised this one as apparently checking against `throws` had not been introduced into typescript until recently.
As long as we are not getting compilation issues on the CI leave it this way. If it fails though, go ahead and remove the method type here 😉